website/content/docs/agent-and-proxy/proxy/caching/static-secret-caching.mdx

```diff
@@ -80,7 +86,7 @@ success `GET` request. If the request is successful, Proxy caches the fact that
 the token was successful in addition to the result. Subsequent requests by the
 same token can then access this secret from the cache instead of Vault.
 
-Vault Proxy uses the [event system](/vault/docs/concepts/events) to keep the
+Vault Proxy uses the [event notification system](/vault/docs/concepts/events) to keep the
```

Comment:
This feature was renamed for 1.16, see also: https://developer.hashicorp.com/vault/docs/v1.16.x/concepts/events